ev3dev2/led.py

```diff
@@ -401,7 +403,8 @@ def animate_police_lights(self, color1, color2, group1='LEFT', group2='RIGHT', s
         def _animate_police_lights():
             self.all_off()
             even = True
-            duration_ms = duration * 1000
+            if duration is not None:
+                duration_ms = duration * 1000
```

Comment:
Can we make this variable `None` rather than leaving it undeclared? Something like 

```python
duration_ms = duration * 1000 if duration is not None else None
```

Reply:
Ok